config/scorecard/bases/config.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
```

Comment:
As said before, it would be cleaner to add scorecard in a separate PR, hence please remove the scorecard config as well